classes/image_types_socfpga.bbclass

```diff
@@ -1,27 +1,25 @@
 inherit image_types
 
-IMAGE_BOOTLOADER ?= "u-boot-socfpga"
 SDCARD_ROOTFS ?= "${DEPLOY_DIR_IMAGE}/${IMAGE_NAME}.rootfs.ext3"
 
 # Boot partition size [in KiB]
 IMAGE_ROOTFS_ALIGNMENT_cyclone5 ?= "2048"
 IMAGE_ROOTFS_ALIGNMENT_arria5 ?= "2048"
-IMAGE_ROOTFS_ALIGNMENT_arria10 ?= "10240"
 
 BOOT_SPACE ?= "102400"
 
 IMAGE_DEPENDS_sdcard = "parted-native:do_populate_sysroot \
                         dosfstools-native:do_populate_sysroot \
                         mtools-native:do_populate_sysroot \
                         virtual/kernel:do_deploy \
-                        ${@d.getVar('IMAGE_BOOTLOADER', True) and d.getVar('IMAGE_BOOTLOADER', True) + ':do_deploy' or ''}"
+                        virtual/bootloader:do_deploy"
 
-SDCARD_GENERATION_COMMAND_cyclone5 = "generate_28nm_sdcard"
+SDCARD_GENERATION_COMMAND_cyclone5 = "generate_28nm_sdcard"               
```

Comment:
sorry...

On Tue, Apr 19, 2016 at 8:29 AM, Khem Raj <notifications@github.com> wrote:

> In classes/image_types_socfpga.bbclass
> <https://github.com/kraj/meta-altera/pull/25#discussion_r60253366>:
>
> >
> > -SDCARD_GENERATION_COMMAND_cyclone5 = "generate_28nm_sdcard"
> > +SDCARD_GENERATION_COMMAND_cyclone5 = "generate_28nm_sdcard"
>
> trailing whitespaces
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kraj/meta-altera/pull/25/files/6ce69405bbfa5e871d71405ff232f30932787f86#r60253366>
>